tools/perf_util/src/main/java/com/msft/send_perf_metrics/App.java

```diff
@@ -98,7 +98,7 @@ static private void loadMetricsIntoMySQL(java.sql.Connection conn, String commit
 		// field name -> json value
 		Map<String, Object> field_mapping = new LinkedHashMap();
 		Set<String> update_on_duplicate_fields =
-			new LinkedHashSet<> (Arrays.asList("AvgTimePerBatch", "Throughput", "StabilizedThroughput", "TotalTime", "AvgCPU", "Memory"));
+			new LinkedHashSet<> (Arrays.asList("AvgTimePerBatch", "Throughput", "StabilizedThroughput", "EndToEndThroughput","TotalTime", "AvgCPU", "Memory"));
```

Comment:
space after "EndToEndThroughput"
nice to see that adding a new field here is simple :)